chunks/lang-go/Dockerfile

```diff
@@ -11,19 +11,20 @@ ENV GO_VERSION=${GO_VERSION}
 ENV GOPATH=$HOME/go-packages
 ENV GOROOT=$HOME/go
 ENV PATH=$GOROOT/bin:$GOPATH/bin:$PATH
-RUN curl -fsSL https://dl.google.com/go/go$GO_VERSION.linux-amd64.tar.gz | tar xzs && \
+RUN curl -fsSL https://dl.google.com/go/go$GO_VERSION.linux-amd64.tar.gz | tar xzs
 # install VS Code Go tools for use with gopls as per https://github.com/golang/vscode-go/blob/master/docs/tools.md
 # also https://github.com/golang/vscode-go/blob/27bbf42a1523cadb19fad21e0f9d7c316b625684/src/goTools.ts#L139
-    go install -v github.com/uudashr/gopkgs/cmd/gopkgs@v2 && \
-    go install -v github.com/ramya-rao-a/go-outline@latest && \
```

Comment:
The old way will reduce docker layers.

Reply:
If you [dive](https://github.com/wagoodman/dive) the `gitpod/workspace-go`, you'll see it gets compressed to a single layer.

Reply:
In other words, normally I totally agree, however, in this case, it helps us more easily see errors when there's trouble with a specific package. In my case, there were a few that had trouble when we were using 1.18.